cpp/iedriver/Browser.cpp

```diff
@@ -34,12 +34,13 @@
 
 namespace webdriver {
 
-Browser::Browser(IWebBrowser2* browser, HWND hwnd, HWND session_handle) : DocumentHost(hwnd, session_handle) {
+Browser::Browser(IWebBrowser2* browser, HWND hwnd, HWND session_handle, bool isEdgeChrome) : DocumentHost(hwnd, session_handle) {
```

Comment:
Should we rename this to isEdgeChromium (similar elsewhere)?

Reply:
Fixed.